sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Drop-application-offering-change-requests.sql

```diff
@@ -0,0 +1,2 @@
+-- DROP Table
+DROP TABLE sims.application_offering_change_requests;
```

Comment:
Are we starting to avoid `IF EXISTS` and `IF NOT EXISTS` ? I remember that dev chat we had.

Reply:
yes we are